### ISSUE TITLE
Enhancement: include sanic in benchmark

### ISSUE THREAD

**Issue by devkral** [litestar-org/litestar#2168]
### Summary

It would be interesting to see how fast this project is compared to sanic. Can you include in the benchmarks sanic?

**Comment by Alc-Alc** [comment]
Hey, just as a side note, in case you missed it, it seems that Sanic is incorporated here: https://github.com/litestar-org/api-performance-tests.

I'm not opposed to including this in the main documentation, I will let the maintainers weigh in on that. I just wanted to let you know.

**Comment by devkral** [comment]
thanks, missed that. Are there results?

**Comment by Alc-Alc** [comment]
> thanks, missed that. Are there results?

I don't think so. In the mean time, it should be fairly trivial to run it though 🙂

**Comment by Goldziher** [comment]
> thanks, missed that. Are there results?

there are in the api-benchmarks repository. You can see there.
